recipe/meta.yaml

```diff
@@ -96,13 +96,10 @@ requirements:
 
 test:
   requires:
-    - python {{ python_min }}  # [unix]
     - pip
   commands:
-    - USER=test spyder -h  # [unix]
```

Comment:
this command should be retained somewhere.

Reply:
Do you mean `USER=test`?

Reply:
Sorry. I misread. I thought it ran pytest